src/test/java/net/syscon/elite/api/resource/impl/ResourceTest.java

```diff
@@ -26,7 +25,6 @@
 import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
 import static org.springframework.core.ResolvableType.forType;
 
-@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
```

Comment:
So I have taken this off as it does not appear to have any impact/effect on the tests.  Shout if you disagree.